### PR TITLE
[v2.7] Update Chinese translation

### PR DESCRIPTION
**Pull request by vickyhella**
- Update Chinese translation for Rancher v2.7

### CODE REVIEW

File: shell/assets/translations/zh-hans.yaml [hunk 1]
```diff
@@ -1808,12 +1808,12 @@ cluster:
     cis:
       server: Server CIS 配置文件
       agent: CIS 配置文件
-      override: 使用 CIS 配置文件时允许覆盖 PSA 默认模板
+      override: 使用 CIS 配置文件时允许覆盖默认 PSA 配置模板
```
Comment (kingsd041) [inline shell/assets/translations/zh-hans.yaml:1811]: 当使用 CIS 配置文件时，允许覆盖默认 PSA 配置模板

**Review by kingsd041 — APPROVED** [review]
LGTM

**Review by niusmallnan — APPROVED** [review]
LGTM

**Comment by richard-cox** [comment]
@vickyhella Please note `master` now tracks the release after 2.7.2 and should contain all 2.7 changes. If the changes are also required for `2.7.2` they should also go in to the `release-2.7.2` branch  (though at this point only critical or incredible safe changes will go in).

**Review by richard-cox — APPROVED** [review]

**Pull request merged**